src/volatile_memory.rs

```diff
@@ -1028,6 +1032,10 @@ where
     /// Returns a pointer to the underlying memory. Mutable accesses performed
     /// using the resulting pointer are not automatically accounted for by the dirty bitmap
     /// tracking functionality.
+    #[deprecated(
+        since = "0.12.0",
```

Comment:
Shouldn't this be 0.13.0 then (as that would be the next release)? Or do you plan to include this in 0.12.0 (in that case the changelog entry is wrong)?

It may slightly confuse people that use 0.12.0 (which does not include the warning), but see the deprecation in the docs.

Reply:
Can we make this part of v0.12.1 instead ? Since this doesn't change the functionality at all ?

Reply:
I thought it may annoy people since a new lint may show up. But https://doc.rust-lang.org/cargo/reference/semver.html#minor-introducing-new-lints suggests a minor bump. So 0.12.1 sounds correct :)